Include/internal/pycore_object.h

```diff
@@ -95,24 +103,63 @@ static inline void _Py_RefcntAdd(PyObject* op, Py_ssize_t n)
 #ifdef Py_REF_DEBUG
     _Py_AddRefTotal(_PyInterpreterState_GET(), n);
 #endif
+#if !defined(Py_NOGIL)
```

Comment:
Is this worth the extra code?
`_Py_RefcntAdd` is only used in `list_repeat` and `tuplerepeat`. Multiplying tuples and lists by ints is hardly a common operation.
Would it perhaps make more sense to remove `_Py_RefcntAdd` and use plain `Py_INCREF` in those functions?